federation/scim/src/main/java/org/keycloak/federation/scim/jpa/ScimResourceMapping.java

```diff
@@ -0,0 +1,88 @@
+package org.keycloak.federation.scim.jpa;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.IdClass;
+import jakarta.persistence.NamedQueries;
+import jakarta.persistence.NamedQuery;
+import jakarta.persistence.Table;
+import org.keycloak.federation.scim.core.service.EntityOnRemoteScimId;
+import org.keycloak.federation.scim.core.service.KeycloakId;
+
+@Entity
+@IdClass(ScimResourceId.class)
+@Table(name = "SCIM_RESOURCE_MAPPING")
+@NamedQueries({
+        @NamedQuery(name = "findById", query = "from ScimResourceMapping where realmId = :realmId and componentId = :componentId and type = :type and id = :id"),
+        @NamedQuery(name = "findByExternalId", query = "from ScimResourceMapping where realmId = :realmId and componentId = :componentId and type = :type and externalId = :id") })
```

Comment:
Looks like named queries from custom JPA entities are not registered and is causing:

```
Caused by: org.hibernate.query.UnknownNamedQueryException: No query named 'findByExternalId'
	at org.hibernate.internal.AbstractSharedSessionContract.buildNamedQuery(AbstractSharedSessionContract.java:1125)
	at org.hibernate.internal.AbstractSharedSessionContract.buildNamedQuery(AbstractSharedSessionContract.java:1130)
	... 42 more
```

Did you run this provider using the Quarkus-based distribution?